src/core-sdk-tasks/ExtractArchiveToDirectory.cs

```diff
@@ -98,16 +106,57 @@ public override bool Execute()
                                 }
                             }
                         }
+                        else
+                        {
+#if NETFRAMEWORK
+                            // Run the base tool, which uses external 'tar' command
+                            retVal = base.Execute();
```

Comment:
Do you happen to know whether this codepath is executed any longer? If it's not, we could just add an NYI exception.

My hunch is it's not.

Reply:
I'm not 100% sure about `NETFRAMEWORK` code-path, but we also use base implementation (and tar command) for full package extraction, which is still needed.